### DISCUSSION TITLE
[Help wanted] Improve docs

### DISCUSSION THREAD

**Discussion by dai-shi** [wakujs/waku#295] [General]

**Comment by himself65** [comment]
I'd like help on this

**Comment by dai-shi** [comment]
Yay! Please go ahead for anything!
(I know @sandren is also motivated to work on it, but I think we can collaborate eventually. So, feel free.)

**Reply by nyomansunima** [reply]
Let me know this

**Reply by dai-shi** [reply]
We need more contributors for the doc contents. There was a small request on the Discord server lately.

**Comment by nyomansunima** [comment]
Hey, it would be cool if we could redesign and make the website cool and easy to navigate.
This will help the new user to understand the framework.

My suggestions:
- Logo
- Icon
- Sections
- Docs
- Styles

References:
- https://nextjs.org
- https://nuxt.com
- https://reflect.app
- https://trigger.dev

It would be cool if we could add animation and interactions and show off the framework.
